contents/shells/bash.md

```diff
@@ -137,15 +137,15 @@ Let us say you want to check the status of process `p` running on your system. T
 
 A more detailed introduction to I/O stream redirection can be found [here](https://www.digitalocean.com/community/tutorials/an-introduction-to-linux-i-o-redirection).
 
-## Why use the shell?
+## Why use the Shell?
```

Comment:
Should the word `use` be capitalised, since it's a verb?
It has been capitalised in other documents:
![image](https://user-images.githubusercontent.com/35621759/76767429-8fb5b780-67d4-11ea-9702-060e4881f5bb.png)

Reply:
Good catch. I will capitalize it.